### PR TITLE
Cut down changelog

### PR DESCRIPTION
**Pull request by treid314**
#### What this PR does
This PR removes the CHANGELOG entries for cortex as the changelog was becoming quite large.

#### Checklist

- [ ] Tests updated
- [ ] Documentation added
- [X] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

**Review by pracucci — APPROVED** [review]
I agree 👍

**Review by pstibrany — APPROVED** [review]
Good idea!

File: CHANGELOG.md [hunk 1]
```diff
@@ -1387,1932 +1387,4 @@ _Changes since Cortex `1.10.0`._
...
-
-* [CHANGE] HA Tracker flags were renamed to provide more clarity #1465
-  * `distributor.accept-ha-labels` is now `distributor.ha-tracker.enable`
-  * `distributor.accept-ha-samples` is now `distributor.ha-tracker.enable-for-all-users`
-  * `ha-tracker.replica` is now `distributor.ha-tracker.replica`
-  * `ha-tracker.cluster` is now `distributor.ha-tracker.cluster`
-* [FEATURE] You can specify "heap ballast" to reduce Go GC Churn #1489
-* [BUGFIX] HA Tracker no longer always makes a request to Consul/Etcd when a request is not from the active replica #1516
-* [BUGFIX] Queries are now correctly cancelled by the query-frontend #1508
+## [Cortex 1.10.0 CHANGELOG](https://github.com/cortexproject/cortex/blob/master/CHANGELOG.md#1100--2021-08-03)
```
Comment (colega) [inline CHANGELOG.md:1390]: Nit, who knows how the future will look like, and whether:
- Cortex will move the file
- Cortex will rename `master` 
- Cortex will move to Gitlab for some reason.

I'd point here to a permalink of current main in Mimir:

```suggestion
## [Cortex 1.10.0 CHANGELOG](https://github.com/grafana/mimir/blob/a13959db5d38ff65c2b7ef52c56331d2f4dbc00c/CHANGELOG.md#cortex-1100--2021-08-03)
```

Comment (pracucci) [reply CHANGELOG.md:1390]: Good suggestion.

Comment (pstibrany) [reply CHANGELOG.md]: When the link changes, we can always fix it :) Personally I think pointing to Cortex changelog would make more sense.

**Review by colega — APPROVED** [review]

**Review by aknuds1 — APPROVED** [review]
:rocket:

**Pull request merged**